core/shared/src/main/scala/org/http4s/ResponseCookie.scala

```diff
@@ -104,7 +104,7 @@ object ResponseCookie {
     import Parser.{char, charIn, failWith, ignoreCase, pure}
     import Rfc2616.Rfc1123Date
     import Rfc5234.digit
-    import Rfc6265.{cookieName, cookieValue}
+    import RelaxedCookies.{cookieName, cookieValue}
```

Comment:
Before this PR, `ResponseCookie` was using a different parser char set than `RequestCookie`, but sharing the same special double-quoted value parsing code. Now that `ResponseCookie` changes to the new relaxed char set which tolerates double quotes, the double-quoted value parsing code can no longer serve both char sets. 
I think the most straightforward and logical fix is what I did here, changing the `ResponseCookie` to use the same char set as `RequestCookie`.